service/controller/resource/cloudconfig/desired.go

```diff
@@ -51,11 +53,84 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
-	var clusterCerts certs.Cluster
+	// TODO
+	//
+	//     * no dependabot
+	//     * ambiguous deps
+	//     *
+	//
```

Comment:
Just my WIP comments on what I noticed. I can drop them but we may want to do something about it. Let's get dependabot setup here. `devctl` can help with that. And then we should take a look at all the different versions we use. At best we use one version per dependency and not three.

Reply:
Agree IMO dependabot would help but I've removed this comment for now.

@tuommaki @nprokopic PTAL and track in Celestial if there is stuff to do here.